v3.0/how-to/maintain/backup-and-restore.md

```diff
@@ -83,6 +83,43 @@ In this command,
 - `-F 64`: means a table is partitioned into chunks and one chunk is 64MB.
 - `--skip-tz-utc`: the purpose of adding this parameter is to ignore the inconsistency of time zone setting between MySQL and the data exporting machine and to disable automatic conversion.
 
+If `mydumper` meeting error likes
+
+```log
+** (mydumper:27528): CRITICAL **: 13:25:09.081: Could not read data from samsung_bench.TEST_TBL: GC life time is shorter than transaction duration, transaction starts at 2019-08-05 21:10:01.451 +0800 CST, GC safe point is 2019-08-05 21:14:53.801 +0800 CST
```

Comment:
`samsung_bench.TEST_TBL` should we use another schema.table name, which is more like an example?